docs/contents/user-docs/irsa.md

```diff
@@ -77,33 +50,61 @@ read -r -d '' TRUST_RELATIONSHIP <<EOF
 EOF
 echo "${TRUST_RELATIONSHIP}" > trust.json
 
-# update variables as needed
+# Update the service name variables as needed
 AWS_SERVICE_NAME='elasticache'
 ACK_CONTROLLER_IAM_ROLE="ack-${AWS_SERVICE_NAME}-controller"
 ACK_CONTROLLER_IAM_ROLE_DESCRIPTION='IRSA role for ACK $AWS_SERVICE_NAME controller deployment on EKS cluster using Helm charts'
```

Comment:
Should be using double quotes, otherwise `$AWS_SERVICE_NAME` will not be substituted.